src/main/scala/cromwell/engine/EngineFunctions.scala

```diff
@@ -1,18 +1,22 @@
 package cromwell.engine
 
 import cromwell.binding.WdlFunctions
-import cromwell.binding.values.{WdlInteger, WdlString, WdlValue}
+import cromwell.binding.values.{WdlFile, WdlInteger, WdlString, WdlValue}
 
 import scala.util.Try
 
 trait EngineFunctions extends WdlFunctions {
   protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger]
   protected def read_string(params: Seq[Try[WdlValue]]): Try[WdlString]
+  protected def stdout(params: Seq[Try[WdlValue]]): Try[WdlFile]
+  protected def stderr(params: Seq[Try[WdlValue]]): Try[WdlFile]
 
   def getFunction(name: String): WdlFunction = {
     name match {
       case "read_int" => read_int
       case "read_string" => read_string
+      case "stdout" => stdout
```

Comment:
So where do we draw the line? Why not say that one layer in has pythonic names? Why not say that values internal to the functions have pythonic names?

I'm being absurd there but these were the sorts of thoughts which led me further down the "scala-fy it" path. Truthfully I don't have a *huge* beef with having very outer edges deviating from convention, although as I said previously if we're doing that I think it should be determined reflectively or something else automagic.